frontend/src/views/c-summary.vue

```diff
@@ -92,7 +94,10 @@
           v-bind:href="getReportIssueEmailLink(errorBlock.errorMessage)"
         )
           span {{ getReportIssueEmailAddress() }}
-      .error-message-box__failed-repo--reason(v-else) {{ errorBlock.errorMessage }}
+      .error-message-box__failed-repo--reason(v-else) {{ errorBlock.errorMessage }}\
+    .error-message-box__show-more-container(v-if="Object.keys(errorMessages).length >= 4")
+      a.error-message-box__show-more(v-if="!errorIsShowingMore" v-on:click="toggleErrorShowMore()") SHOW MORE...
+      a.error-message-box__show-more(v-else v-on:click="toggleErrorShowMore()") SHOW LESS...
```

Comment:
CI seem to have indicated this area as a potential violation of certain linting rules, might warrant further investigation to know what went wrong!